Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -141,25 +141,33 @@ LayoutRepainter::CheckForRepaint SVGRenderSupport::checkForSVGRepaintDuringLayou
     return LayoutRepainter::CheckForRepaint::Yes;
 }
 
-auto SVGRenderSupport::computeContainerBoundingBoxes(const RenderElement& container, RepaintRectCalculation repaintRectCalculation) -> ContainerBoundingBoxes
+static bool hasValidBoundingBoxForContainer(const RenderObject& object)
 {
-    ContainerBoundingBoxes result;
+    if (auto* svgShape = dynamicDowncast<LegacyRenderSVGShape>(object))
+        return svgShape && !svgShape->isRenderingDisabled();
```

Comment:
I don’t think our local variable name has to be this specific. Can just be a sensible word:
```suggestion
    if (auto* shape = dynamicDowncast<LegacyRenderSVGShape>(object))
        return shape && !shape->isRenderingDisabled();
```

Reply:
Done.